sources/packages/api/src-sql/SINValidations/Create-sin-validations.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE IF NOT EXISTS sims.sin_validations (
+  id SERIAL PRIMARY KEY,
+  date_sent TIMESTAMP WITH TIME ZONE,
+  date_received TIMESTAMP WITH TIME ZONE,
+  file_sent VARCHAR(50),
+  file_received VARCHAR(50),
+  given_name_sent VARCHAR(50),
+  surname_sent VARCHAR(50),
+  dob_sent Date,
+  request_status_code CHAR(2),
+  match_status_code CHAR(2),
+  sin_match_status_code CHAR(2),
+  surname_match_status_code CHAR(2),
+  given_name_match_status_code CHAR(2),
+  dob_match_status_code CHAR(2),
+  -- Audit columns
+  created_at timestamp with time zone NOT NULL DEFAULT now(),
+  updated_at timestamp with time zone NOT NULL DEFAULT now(),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+  SET
+    NULL,
+    modifier INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+  SET
+    NULL
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.sin_validations IS 'SIN Validations that must be performed for the user.';
```

Comment:
Should we have the user id associated with this table?

Reply:
Yes you are right, I am adding it as per our discussion